server/config/config.go

```diff
@@ -204,7 +204,7 @@ const (
 	defaultCompactionMode          = "periodic"
 	defaultAutoCompactionRetention = "1h"
 	defaultQuotaBackendBytes       = typeutil.ByteSize(8 * 1024 * 1024 * 1024) // 8GB
-	defaultMaxRequestBytes         = uint(1.5 * 1024 * 1024)                   // 1.5MB
+	defaultMaxRequestBytes         = uint(150 * 1024 * 1024)                   // 150MB
```

Comment:
Please add a comment about why we choose the value.

Reply:
done